superset-embedded-sdk/src/index.ts

```diff
@@ -149,7 +154,7 @@ export async function embedDashboard({
         resolve(new Switchboard({ port: ourPort, name: 'superset-embedded-sdk', debug }));
       });
 
-      iframe.src = `${supersetDomain}/embedded/${id}${dashboardConfig}${filterConfigUrlParams}`;
+      iframe.src = `${supersetDomain}/embedded/${id}${dashboardConfig}${filterConfigUrlParams}${extraUrlParams}`;
```

Comment:
There might be a conflict between keys in filterConfigUrlParams and extraUrlParams. It would be better to design a unique key-value pair and override its value sequentially, something like

```suggestion
      const getUniqueUrlParams = (urlParams: string[]) =>
        new URLSearchParams(
          Object.fromEntries(
            urlParams.map(param => [...new URLSearchParams(param)]).flat(1),
          ),
        ).toString();
      const urlParams = getUniqueUrlParams([dashboardConfig, filterConfigUrlParams, extraUrlParams]);
      iframe.src = `${supersetDomain}/embedded/${id}${urlParams ? `?${urlParams}` : ''}`;
```

Reply:
Good suggestion, @justinpark, thanks! I simplified the computation of all three url params categories, please review.